src/protean/core/entity.py

```diff
@@ -118,6 +123,11 @@ def _create_id_field(new_class):
         new_class._meta.declared_fields['id'] = id_field
         new_class._meta.id_field = id_field
 
+    def _load_attributes(new_class):
```

Comment:
Why are we saving this again in another variable?

Reply:
We need a different variable compared to `declared_fields` because what is saved in the DB may be different from what is declared. For example, the `Dog` class may have a declared field of `owner = field.Reference(Human)`, but the actual attribute saved is `owner_id`.

And this is where it comes handy: https://github.com/proteanhq/protean/pull/86/files#diff-6bad0690803c1b00ee464f016cd0609cR300